src/main/scala/com/fortysevendeg/scala/android/modules/validators/DummyFormValidation.scala

```diff
@@ -0,0 +1,28 @@
+package com.fortysevendeg.scala.android.modules.validators
+
+import com.fortysevendeg.scala.android.modules.utils.validation.Validator
+import com.fortysevendeg.scala.android.ui.validateforms.FormModel
+
+import scalaz.Validation.FlatMap._
+
+trait DummyFormValidation extends BaseValidation {
+
+  val validationDescriptionAction = "saving form"
+
+  def isValidNameField: Validator[FormModel] =
+    input => hasRequiredField("name", input.name.getStringValue, validationDescriptionAction)(input)
+
+  def isValidEmailField: Validator[FormModel] =
+    input =>
+      for {
+        isFilled <- hasRequiredField("email", input.email.getStringValue, validationDescriptionAction)(input)
```

Comment:
Good catch. I agree with you, I don't like hardcoded strings in the code. Let me think about it.